phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -28,9 +30,12 @@
     @Value("${application.name}")
     private String applicationName;
 
+    @Autowired
```

Comment:
It's now parallel to the ViewsConfiguration. Is that ok (because the theme belongs to the views somehow)?